src/asciidoc/docs/_includes/startGuide1.svelte

```diff
@@ -0,0 +1,61 @@
+<script>
```

Comment:
Just another general note -- similar to the angular template, this one has a bunch of unused files in the .koji folder.